.github/workflows/tests.yaml

```diff
@@ -614,7 +614,7 @@ jobs:
                      --factoryreset \
                   '
             - name: Run Pairing Address-PaseOnly Test
-              timeout-minutes: 10
+              timeout-minutes: 15
```

Comment:
I don't think this is the problem. Had the same attempt, but looking at the history:

https://github.com/project-chip/connectedhomeip/actions/runs/4299785507/jobs/7495345664

They finish in 1s when they do. If they take 10m, I would rather believe this is a bug/deadlock/hang than a "this needs more time"

Reply:
Yes I agree I should check that out, thanks